src/destructors.md

```diff
@@ -1,11 +1,15 @@
 # Destructors
+r[destructors]
```

Comment:
In another PR, where the `r[..]` had the same name as the primary header, it was elided.  Do we want to elide these or include them?

Reply:
I'd like to have the rendered syntax there, but link-check says no.